athenet/layers/__init__.py

```diff
@@ -1,8 +1,12 @@
 from layer import *
+
 from activation import *
+from concat import *
 from conv import *
 from dropout import *
 from fully_connected import *
 from norm import *
 from pool import *
 from softmax import *
+
+from inception import *
```

Comment:
Consider changing import * to importing specific objects. It would be safer in view of any changes in the future. Generally me @heurezjusz and @martacieslak decided to resign from using import *.